.github/workflows/ci.yml

```diff
@@ -145,6 +145,12 @@ jobs:
     - name: Build crate
       run: cargo $CMD build $OPTIONS
 
+    - name: Test dpi crate
+      if: >
+        contains(matrix.platform.target, 'darwin') &&
```

Comment:
Why only on MacOS?

Reply:
No platform specific code.

Reply:
Why are we testing on MacOS and not on Linux?

Reply:
it has only one job on CI, and linux has like 4.

Reply:
It does seem kind of weird to test on macOS (where CI hours are expensive) and not on Linux (where CI is cheap). Maybe rig it so it only runs on Linux with both features enabled?

Reply:
Would also be in favor of that.

Reply:
>macOS (where CI hours are expensive) and not on Linux (where CI is cheap).

it's run anyway. I don't see any issue with that. it's also not like we're paying for any of that and 1 extra step is like nothing.

Like I can make it more complex (because you need way more checking), but why?

Also, if you want, you can change it yourself.

Reply:
> Also, if you want, you can change it yourself.

Fixed!